packages/consumption/test/modules/requests/testHelpers/TestObjectFactory.ts

```diff
@@ -162,6 +164,102 @@ export class TestObjectFactory {
         });
     }
 
+    public static createRelationshipToDeletedPeer(properties?: Partial<IRelationship>): Relationship {
+        return Relationship.from({
+            id: properties?.id ?? CoreId.from("REL1"),
+            peer:
+                properties?.peer ??
+                Identity.from({
+                    address: CoreAddress.from("did:e:a-domain:dids:anidentity"),
+                    publicKey: CryptoSignaturePublicKey.from({
+                        algorithm: CryptoSignatureAlgorithm.ECDSA_ED25519,
+                        publicKey: CoreBuffer.from("L1sPFQgS5CxgGs1ejBcWCQLCpeFXbRc1TQnSpuHQqDQ")
+                    })
+                }),
+            peerDeletionInfo:
+                properties?.peerDeletionInfo ??
+                PeerDeletionInfo.from({
+                    deletionStatus: PeerDeletionStatus.Deleted,
+                    deletionDate: CoreDate.from("2022-01-03T00:00:00.000Z")
+                }),
+            status: properties?.status ?? RelationshipStatus.Active,
```

Comment:
Is the Relationship not set to the 'DeletionProposed' status after the peer has deleted themself?

Reply:
Yes when the Backbone has finished with the implementation, so I changed it now like it will be in the future...